### PR TITLE
D sminorcorrections

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
DS 11-18 minor corrections

### CODE REVIEW

File: DayrSuryan/DSEthiop12.xml [hunk 1]
```diff
@@ -143,12 +145,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         ወእኂዝ፡ ለዓለመ፡ ዓለም፡ አሚን። ። ። ። ።
                      </explicit>
                      <note>The text seems to have been copied with omissions and transposition of some passages.
-                      The incipit <foreign xml:lang="gez">ዘነገሮሙ፡ እግዚእነ፡...</foreign> is attested in some manuscripts.
+                      The incipit <foreign xml:lang="gez">ዘነገሮሙ፡ እግዚእነ፡...</foreign> is attested in some manuscripts only.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:148]: is attested only in some manuscripts

File: DayrSuryan/DSEthiop15.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                      <incipit xml:lang="gez">
                         <locus from="3ra" to="3rb"/>
-                        ላሎተ፡ ዕፅ፡ እምፃማ<hi rend="rubric">፨</hi> ፈያታውያ<add place="above">ን፡</add>
+                        <supplied reason="undefined">ጽ</supplied>ላሎተ፡ ዕፅ፡ እምፃማ<hi rend="rubric">፨</hi> ፈያታውያ<add place="above">ን፡</add>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop15.xml:49]: resp="DN"

File: DayrSuryan/DSEthiop12.xml [hunk 3]
```diff
@@ -119,7 +119,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         አርዳኢሁ፡ ድኅረ፡  ትንሳ<del rend="erasure" unit="chars" extent="1"/>ኤሁ፡
                            ቅድመ፡ <hi rend="rubric">እርገቱ፡ በስጋ፡</hi> ለመድ<add place="above">ሀ</add>ኒነ፡
                            ዘመላ፡ ሰማያተ፡ ወምድረ፡ ቅድሳተ፡ ስብሃቲሁ፡<hi rend="rubric">ጸሎቱ፡ ወበረ<supplied reason="omitted" resp="DR">ከ</supplied>ቱ፡
-                           የሃሉ፡ ምስለ፡</hi>
+                           የሃሉ፡ ምስለ፡<gap reason="omitted" resp="DN"/>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:122]: Is it possible that there is a problem here with the opening and closing of hi? Could you check again in the manuscript?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: DayrSuryan/DSEthiop12.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop12.xml:148]: ok

File: DayrSuryan/DSEthiop12.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop12.xml:122]: done

File: DayrSuryan/DSEthiop15.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop15.xml:49]: ok

**Review by thea-m — APPROVED** [review]

**Pull request merged**